ui/tests/unit/utils/chart-helpers-test.js

```diff
@@ -37,6 +37,7 @@ module('Unit | Utility | chart-helpers', function () {
     ];
     const testArray2 = [
       { label: 'foo', value: undefined },
+      { label: 'bar', value: 0 },
```

Comment:
non-number values are now excluded from the average method, so added this to the array so the test would still pass